core/docker/build.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF 1>&2
+Usage: $0 [-h] [-a <ARCHITECTURES>] [-v <VERSION>]
+Builds the Trino Docker image
+
+-h       Display help
+-a       Build the specified comma-separated architectures, defaults to amd64,arm64
+-v       Build the specified Trino version, downloads all required artifacts
+EOF
+}
+
+ARCHITECTURES=(amd64 arm64)
+TRINO_VERSION=
+
+while getopts ":a:h:v:" o; do
+    case "${o}" in
+        a)
+            IFS=, read -ra ARCHITECTURES <<< "$OPTARG"
+            ;;
+        v)
+            TRINO_VERSION=${OPTARG}
+            ;;
+        h)
+            usage
+            exit 0
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+shift $((OPTIND - 1))
+
+SOURCE_DIR="../.."
+WORK_DIR="$(mktemp -d)"
+
+# Retrieve the script directory.
+SCRIPT_DIR="${BASH_SOURCE%/*}"
+cd "${SCRIPT_DIR}" || exit 2
+
+pushd ${SOURCE_DIR}
```

Comment:
is it avoidable? what would it take to have constant work dir?

Reply:
I think `mvnw` doesn't like to be called from a relative path.

Reply:
right

Reply:
comment?

Reply:
Maybe `./mvnw -f path/to/root/pom.xml` should work?

Reply:
That works, thanks! I removed `pushd`.